### ISSUE TITLE
Coming messages speed

### ISSUE THREAD

**Issue by Jamuzb** [vysheng/tg#543]
Hi.

I have such issue:
I run on daemon mode and some times. coming messages speed is being very slow. 
On logs I can see:
** Hole in seq (seq = 444, cur_seq = 435)
** Hole in seq (seq = 445, cur_seq = 435)
** Hole in seq (seq = 446, cur_seq = 435)
................................................................

** Hole in seq (seq = 1523, cur_seq = 435)
** Hole in seq (seq = 1524, cur_seq = 435)
** Hole in seq (seq = 1525, cur_seq = 435)

cur_seq is not increasing and I'm receiving old messages with very low speed.

But if I reuthorize(deleting /root/.telegram-cli folder) sequnces are starting from 1 and old messages are not coming.
What is the trick that it is working on reauthorization. is it possible to not read old mesages without reauthorization but with only restart of cli..

If anyone can help me I can show old configurations(lua, running params)
Running params: 
./bin/telegram-cli -vvvvRC -E -f -N  -k perfectum.pub -W -s tg.lua -dL tg.log -l 3 -P 1234

Vysheng do you have any idea about the issue

Thanks.

**Comment by vysheng** [comment]
Maybe you have some very active bot?

**Comment by Jamuzb** [comment]
Yes. You are right. My Bot is very active. Telegram support didn't answered yet.
My bot is in more than 5K groups.
